DataFormats/simulation/include/SimulationDataFormat/MCCompLabel.h

```diff
@@ -120,6 +120,7 @@ class MCCompLabel
   int getTrackIDSigned() const { return isFake() ? -getTrackID() : getTrackID(); }
   int getEventID() const { return (mLabel >> nbitsTrackID) & maskEvID; }
   int getSourceID() const { return (mLabel >> (nbitsTrackID + nbitsEvID)) & maskSrcID; }
+  ULong64_t getTrackEventSourceID() const { return static_cast<ULong64_t>(mLabel & maskFull); }
```

Comment:
@sawenzel : I need this to extract a unique identifier of the label for external comparisons. This not consider additional flags or e.g. a weight field that might be added later, so I added an extra getter for the combined track/event/source ID if ok with you.